src/components/DoctorCard/PageInfo.js

```diff
@@ -57,7 +57,11 @@ const PageInfo = ({ doctor }) => {
             <Typography component="div" variant="body1">
               <Icons.Icon name="Phone" />
             </Typography>
-            <Shared.ConditionalLink to={doctor?.phone && `tel:${doctor.phone}`} variant="body1">
+            <Shared.ConditionalLink
+              to={doctor?.phone && `tel:${doctor.phone}`}
+              self
+              variant="body1"
+            >
```

Comment:
If `doctor?.phone` is `undefined` or falsy, what will happen here? I would propose the following change (conditionally render the component alltogether):

```suggestion
            {doctor?.phone && (<Shared.ConditionalLink
              to={`tel:${doctor.phone}`}
              self
              variant="body1"
            >{doctor.phone}
            </Shared.ConditionalLink>)}
```

Reply:
Actually I don't need optional chaining at all. A few lines higher I am alredy condtitionally rendering based on doctor.phone.